packages/client-ts/src/utils.ts

```diff
@@ -75,15 +75,12 @@ export function explodeVeridaUri(uri: string): FetchUriParams {
 
 /**
  * Fetch the data accessible from a Verida URI
- * 
+ *
  * @param uri Verida URI of the record to access
  * @param context An existing context used to open the external database
- * @returns 
+ * @returns
  */
-export async function fetchVeridaUri(
-	uri: string,
-	context: Context
-): Promise<string> {
+export async function fetchVeridaUri(uri: string, context: any): Promise<string> {
```

Comment:
This method should return `Promise<object>`.

Reply:
the function has been modified.